tests/testing_tests/test_causal_test_case.py

```diff
@@ -118,7 +118,7 @@ def test_execute_test_observational_causal_forest_estimator(self):
             self.df,
         )
         causal_test_result = self.causal_test_case.execute_test(estimation_model, self.data_collector)
-        self.assertAlmostEqual(causal_test_result.test_value.value, 4, delta=1)
+        self.assertAlmostEqual(causal_test_result.test_value.value[0], 4, delta=1)
```

Comment:
Just a readability thing, but would it be worth doing something like `self.assertAlmostEqual(causal_test_result.test_value.value, pd.Series(4), delta=1)` instead to show that we expect the result to be a single element series?

Reply:
I really like that approach! I'll go through this afternoon and update any cases to that 👍

Reply:
I hadn't noticed that these tests use `self.assertAlmostEqual` which can't compare `pd.Series` objects. 

I think any attempt to add the semantics of the test result being a `pd.Series`, would add too much complexity to the test.

Reply:
We could just round the values and then use `self.assertEqual`?

Reply:
I don't think you can make any assertions directly on pandas objects using only pytest. I think we would have to employ something such as [this](https://pandas.pydata.org/docs/reference/api/pandas.testing.assert_series_equal.html).

Reply:
That's quite nice, though, I think. The last one I had to do, I just did `assertTrue(all(x1==x2))`, which doesn't provide a nice error message